kernel/generic/src/ipc/sysipc.c

```diff
@@ -436,7 +436,7 @@ sys_errno_t sys_ipc_call_async_slow(cap_phone_handle_t handle, ipc_data_t *data,
 		return ENOMEM;
 	}
 
-	errno_t rc = copy_from_uspace(&call->data.args, &data->args,
+	errno_t rc = copy_from_uspace(&call->data.args, data + offsetof(ipc_data_t, args),
```

Comment:
Hm, this is also unfortunate that we can't no longer use the compiler to compute the member address.